lib/node/neutrino.js

```diff
@@ -0,0 +1,324 @@
+/*!
+ * neutrino.js - spv node for bcoin
+ * Copyright (c) 2014-2015, Fedor Indutny (MIT License)
+ * Copyright (c) 2014-2017, Christopher Jeffrey (MIT License).
```

Comment:
lets just go with 2023 the bcoin developers ;-) Fedor and JJ didn't help at all with this module!

Reply:
Okay! :)